src/Integration.Vsix/InfoBar/InfoBarManager.cs

```diff
@@ -144,12 +161,17 @@ private static bool TryCreateInfoBarUI(IVsInfoBarUIFactory infoBarUIFactory, IVs
             return uiElement != null;
         }
 
-        private static InfoBarModel CreateModel(string message, string buttonText, SonarLintImageMoniker imageMoniker)
+        private static InfoBarModel CreateModel(string message, IEnumerable<string> buttonTexts, ButtonStyle buttonStyle, SonarLintImageMoniker imageMoniker)
         {
-            var vsImageMoniker = new ImageMoniker {Guid = imageMoniker.Guid, Id = imageMoniker.Id};
+            var vsImageMoniker = new ImageMoniker { Guid = imageMoniker.Guid, Id = imageMoniker.Id };
+
+            var actionItems = buttonTexts.Select(x =>
+                buttonStyle == ButtonStyle.Button
+                    ? (IVsInfoBarActionItem) new InfoBarButton(x)
+                    : new InfoBarHyperlink(x));
 
             return new InfoBarModel(message,
-                actionItems: buttonText != null ? new[] { new InfoBarButton(buttonText) } : new IVsInfoBarActionItem[0],
```

Comment:
I'm guessing we didn't previously have any callers that supplied `null`?

Reply:
Technically if you called `AttachInfoBar` instead of `AttachInfoBarWithButton` then this could would be called with null.
But no one calls `AttachInfoBar` in the code, so no :)